frontend/src/core/comments/components/AddComment.css

```diff
@@ -17,7 +17,35 @@
     line-height: 24px;
     overflow: auto;
     padding: 6px;
-    resize: none;
+    word-break: break-all;
+}
+
+.comments-list .add-comment .comment-editor span {
+    vertical-align: baseline !important;
+}
+
+.mention-list {
+    top: -9999px;
+    left: -9999px;
+    position: absolute;
```

Comment:
We'll need to set the parent element to relative, otherwise the mention list is fixed on scroll.

Reply:
Since this is inside a `<Portal />`  that is going to take a bit of research as it sits outside of the DOM hierarchy. 

https://reactjs.org/docs/portals.html

Reply:
Also need to work out how reposition it to hover on top when it hits the bottom of the view

![Screen Shot 2020-04-27 at 9 53 54 AM](https://user-images.githubusercontent.com/19564687/80398962-4ba2e180-886d-11ea-8756-1f8f06ede42d.png)

Reply:
That's definitely another UX glitch - if you have existing translation comments, the popup sometimes opens so low in the screen that you don't even see a pixel of it.

Reply:
I randomly bumped into this, maybe it'll help:
https://blog.logrocket.com/controlling-tooltips-pop-up-menus-using-compound-components-in-react-ccedc15c7526/

Reply:
I just pushed changes that address both of these issues.  Pretty sure they can be improved but wanted to see if I was even on the right track 😄